tests/test_cnn.py

```diff
@@ -566,6 +560,7 @@ def test_find_duplicates_verbose_false(capsys):
 
 def test_scores_saving(cnn):
     save_file = 'myduplicates.json'
+    cnn = CNN(verbose=False)
```

Comment:
Not sure why this should fix the problem, but if it allows azure pipelines to pass, it can be left in.

In my local system this test still fails with error: 
> 'E   IndexError: index 5 is out of bounds for axis 0 with size 5'